docs/hosts/quickstart.md

```diff
@@ -5,54 +5,76 @@ sidebar_position: 2
-            - `peers` You will need the peer's IP_ADDRESS and PEER_ID to complete the connection string: ['/ipv4/\<IP_ADDRESS\>/tcp/9171/p2p/\<PEER_ID\>']
-        - `listen_addr` - the default value of "/ip4/0.0.0.0/tcp/0" should be sufficient when running locally. If running in a container, you'll want to manually choose a port "/ip4/0.0.0.0/tcp/\<your port here\>" so that you can expose that port in your container.
+The Host Client reads from `config.yaml`, which comes with sensible defaults.
+The only field you need to set is **defradb.keyring_secret** which can alternatively be set with the following command in the terminal window.
 
-- `store`
-    - `path` - with this, you can change where your Host's embedded defra instance stores data. In general, the default should be fine.
+```bash
+export DEFRA_KEYRING_SECRET=<make_a_password>
+```
 
-- `shinzo` *Very important - if not done correctly your Host will not receive View definitions and will have nothing to Host*
-    - `web_socket_url` - here you must provide a valid websocket url for Shinzohub, we will use this to subscribe to new View events emitted by Shinzohub. At time of writing, the Host will begin hosting any View it receives via this event subscription
+### Key Fields
 
-- `logger`
-    - `development` - leave this as true if doing any kind of testing. Otherwise, make this false - this will quiet the Defra logs which can quickly fill up your memory if you aren't careful.
+* **defradb.url** – API endpoint of your local DefraDB node. Defaults work for most setups.
+* **defradb.keyring_secret** – Requires a secret to generate your private keys. 
+* **p2p.bootstrap_peers** – Seed peers for joining the Shinzo network. Defaults include a reliable bootstrap peer.
+* **p2p.listen_addr** – Default is suitable for local runs. Override when containerizing.
+* **store.path** – Directory where local DefraDB data is stored.
+* **shinzo.web_socket_url** – Defaults to a hosted ShinzoHub node. Only change if connecting to a different node.
+* **logger.development** – Set to `false` for production.
+* **host.lens_registry_path** – Where received WASM lens files are stored.
 
-- `host`
-    - `lens_registry_path` - with this, you can change where your Host stores the Lens wasm files it receives. In general, the default should be fine.
+### Default Behavior
 
-## Running Locally
+The included `config.yaml` is ready for most local development workflows. You should only need to modify peer settings or storage paths for advanced setups.
 
-Once configured, you can run the Host client app locally with
+---
+
+## 4. Running the Host
+
+To start the Host Client:
```

Comment:
@akc2267 Can we add a note here clarifying that the localhost WebSocket URL is only for users running a local ShinzoHub? We should also mention that this value can be replaced with the deployed ShinzoHub endpoint when not running locally.